src/modules/datepicker.js

```diff
@@ -0,0 +1,33 @@
+import "../../node_modules/@chenfengyuan/datepicker/dist/datepicker.min.js";
+import "../../node_modules/@chenfengyuan/datepicker/dist/datepicker.min.css";
+
+const today = new Date();
+const startDay = "01/01/2018";
+
+const startOptions = {
+  date: startDay,
+  autoHide: true,
+  startDate: startDay,
+  endDate: today,
+  autoPick: true
+};
+
+const endOptions = {
+  date: today,
+  autoHide: true,
+  startDate: startDay,
+  endDate: today,
+  autoPick: true
+};
+
+export function convertDateStringToUnix(dateString) {
```

Comment:
thanks for walking me through this. `Date()`  assumes `dateString` is in local time, and applies a timezone offset accordingly.